README.md

```diff
@@ -1,5 +1,14 @@
-# Power Grid
-Power grid is an easy to use yet a powerful tool to readily build a cross-platform CSS Grid system.  
+# Powergrid
+
+![screen](screens/shot1.PNG)
+
```

Comment:
Will be able to see this image after commit, so if required we change after verifying it.

Reply:
I think we can use some sort of service that would generate screens from the home page https://zs.github.com/powergrid

Reply:
E.g. https://www.url2png.com/

Reply:
Tried this services as well as some other services also, but its not taking screenshots as required. And also most of them are paid services.